src/compiler/objective_c_plugin.cc

```diff
@@ -82,10 +81,12 @@ class ObjectiveCGrpcGenerator : public grpc::protobuf::compiler::CodeGenerator {
         declarations += grpc_objective_c_generator::GetHeader(service);
       }
 
+      ::grpc::string nonNullBegin = "\nNS_ASSUME_NONNULL_BEGIN\n\n";
```

Comment:
Sorry for the long response time, I was very busy :) Very good proposal, I am not that much into C++ so I am not familiar with all coding styles :) I've changed it in 1843ccb